docs/topics/js/js-to-kotlin-interop.md

```diff
@@ -111,13 +112,49 @@ It can also be applied on file-level using `@file:JsExport`.
 To resolve ambiguities in exports (like overloads for functions with the same name), you can use the `@JsExport`
 annotation together with `@JsName` to specify the names for the generated and exported functions.
 
-The `@JsExport` annotation is available in the current default compiler backend and the new [IR compiler backend](js-ir-compiler.md).
-If you are targeting the IR compiler backend, you **must** use the `@JsExport` annotation to make your functions visible
-from Kotlin in the first place.
+In the current [IR compiler backend](js-ir-compiler.md), the `@JsExport` annotation is the only way to make your functions
+visible from Kotlin in the first place.
 
 For multiplatform projects, `@JsExport` is available in common code as well. It only has an effect when compiling for
 the JavaScript target, and allows you to also export Kotlin declarations that are not platform specific.
 
+### @JsStatic
+
+> This feature is [Experimental](components-stability.md#stability-levels-explained). It may be dropped or changed at any time.
+> Use it only for evaluation purposes. We would appreciate your feedback on it in [YouTrack](https://youtrack.jetbrains.com/issue/KT-18891/JS-provide-a-way-to-declare-static-members-JsStatic).
+>
+{type="warning"}
+
+The `@JsStatic` annotation instructs the compiler to generate additional static methods for the target declaration.
+This helps you use static members from your Kotlin code directly in JavaScript.
+
+You can apply the `@JsStatic` annotation to functions defined in named objects, as well as in companion objects declared
+inside classes and interfaces. The compiler generates both a static method of the object and an instance method in the
+object itself. For example:
+
+```kotlin
+// Kotlin
+class C {
```

Comment:
Did you try formatting? I tested formatting in IntelliJ and got a bit different result:

```
class C {
    companion object {
        @JsStatic
        fun callStatic() {
        }
        fun callNonStatic() {}
    }
}
```